src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-customtag-update.js

```diff
@@ -182,7 +184,7 @@ export default class EzBtnCustomTagUpdate extends EzWidgetButton {
             this.execCommand();
         }
 
-        const widget = this.getWidget();
+        const widget = this.getWidget() || this.widget;
```

Comment:
Can you tell me more about it? You're setting `this.widget` value in the constructor. When the output of `this.getWidget()` will change its value?

Reply:
The output of `this.getWidget()` will be null when we will lose focus in edit custom tag. This happens when we are editing custom tag and use cmd+a to select text in the input.